src/models/weatherData/wind.js

```diff
@@ -0,0 +1,8 @@
+export default class Wind {
+  constructor(data) {
```

Comment:
`inSightWindData`?

Reply:
I've changed it to inSightWeatherData because we're adding feeding in all of it to the wind class